system/netdata.conf

```diff
@@ -19,6 +19,10 @@
     # by default do not expose the netdata port
     bind to = localhost
 
+    # some defaults to run netdata with least priority
+    process scheduling policy = idle
+    OOM score = 1000
```

Comment:
What's the rationale for changing these values from what they were before? (_I'm not 100% sure why they were both `keep` before..._)

Reply:
I have not changed them. They are the defaults used by netdata as well as netdata.service.in

> CPUSchedulingPolicy=idle
> OOMScoreAdjust=1000

https://docs.netdata.cloud/daemon/#netdata-process-scheduling-policy
https://docs.netdata.cloud/daemon/#oom-score

_keep_ actually did nothing in current default case.

So I have added them just as a guideline, in case user wants to change the default.

Reply:
Ahh no worries :) For _some_ reason I thought they were changed from the ones passed in via `ExecStart` 👍